addons/udes_stock/views/stock_picking.xml

```diff
@@ -193,6 +193,10 @@
                 <field name="u_location_category_id"/>
             </xpath>
 
+            <xpath expr="//field[@name='batch_id']" position="after">
+                <field name="u_batch_user_id"/>
```

Comment:
Do Product want this adding to the form view as well as the tree view?

Reply:
Spoke to Nuno, he said he doesn't want it.